processor/processor.go

```diff
@@ -420,8 +434,13 @@ var (
 )
 
 func loadConfig() {
+	config.RegisterBoolConfigVariable(true, &enablePipelining, false, "Processor.enablePipelining")
+	config.RegisterIntConfigVariable(1, &pipelineBufferedItems, true, 1, "Processor.pipelineBufferedItems")
```

Comment:
I see this is defined as hot reloadable. It doesn't look straight forward hot reloadable in code below. The only way for the new value to reflect is by pausing and resuming the processor? Just for clarification.

Reply:
If that is indeed the case, we should add a comment?

Reply:
Good catch it should not be hot reloadable